src/states/Game.js

```diff
@@ -1,49 +1,93 @@
 /* globals __DEV__ */
 import Phaser from 'phaser';
-//import Mushroom from '../sprites/Mushroom';
+import StaticAsset from '../sprites/staticAsset';
+import Player from '../sprites/player';
 
 export default class extends Phaser.State {
   init() {}
   preload() {}
 
   create() {
     //create world
-    this.game.physics.startSystem(Phaser.Physics.ARCADE);
+    this.physics.startSystem(Phaser.Physics.ARCADE);
 
-    this.game.add.sprite(0, 0, 'star');
+    this.add.sprite(0, 0, 'star');
 
-    const platforms = this.game.add.group();
-    platforms.enableBody = true;
-    const ground = platforms.create(0, this.game.world.height - 64, 'platform');
-    ground.scale.setTo(2, 2);
-    ground.body.imovable = true;
-    let ledge = platforms.create(100, 200, 'platform');
-    ledge.scale.setTo(0.4, 0.7);
-    ledge.body.immovable = true;
+    this.ledge = new StaticAsset({
```

Comment:
This _add-a-ledge-and-set-some-properties_ code is repeated 3 times. Perhaps it can be abstracted into a function which you call three times instead?